_posts/2020-10-05-Evidence_locker_overview.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+category: Architecture
+title: Evidence locker overview
+---
+
+At the core of the Auditree framework is a [Git][] repository known as the
+evidence locker.  Each Auditree execution environment requires a dedicated
+evidence locker in order to execute properly.  Serving as the Auditree
+data store, the evidence locker houses evidence gathered by fetchers,
+evidence generated by checks and/or evidence planted by the [Auditree Plant][]
+tool.
+
+## Local vs. remote
+
+The reasoning behind our choice of [Git][] as the Auditree data store is
+outlined nicely in our [previous post][] but in a nutshell, a local evidence
+locker works hand-in-hand with a remote evidence locker hosted by any Git
+repository hosting service (GitHub, GitLab, ...).  Although it is possible to
+execute Auditree with only a local evidence locker, a typical implementation of
+an Auditree execution environment will pull down a local clone of a remote
+evidence locker, perform Auditree operations, store the results of those
+operations as evidence in the clone (local locker) and once complete, push
+changes back to the remote evidence locker.
+
+## Organization
+
+An evidence locker is organized into (up to) four top level folders:
```

Comment:
"We namespace evidence into four top level folders within the locker"? It's more than just organising - there's a meaning to the names, and it's enforced by the code.

Maybe mention that the framework manages putting files in the right place?

Reply:
Done in https://github.com/auditree/auditree.github.io/pull/6/commits/406f0ed0a3b7b1b75e25c945c7f52be953d95e0f